src/main/java/net/imagej/ops/coloc/ColocUtil.java

```diff
@@ -51,4 +53,15 @@ public static boolean sameIterationOrder(Iterable<?> i1, Iterable<?> i2) {
 			IterableInterval<?> ii2 = (IterableInterval<?>) i2;
 			return Intervals.equalDimensions(ii1, ii2) && Util.equalIterationOrder(ii1, ii2);
 		}
+
+	/** Fisher-Yates shuffle. */
+	public static void shuffle(int[] array, Random rnd) {
```

Comment:
Would it make sense to put this into a utility class somewhere in SciJava? Almost the same is use in SCIFIO's `FakeFormat` [here](https://github.com/scifio/scifio/blob/9562ad0a6c4132bc3881d0483b71d1f25eb6ad1a/src/main/java/io/scif/formats/FakeFormat.java#L830-L839).

(This comment shouldn't block merging this PR, of course.)

Reply:
@ctrueden ... thoughts on this one?  I defer to you ...